app/controllers/items_controller.rb

```diff
@@ -55,27 +49,21 @@ def destroy
     redirect_to items_path
   end
 
-  def new_storage_collection
-    @storages.each do |storage|
-      @row_collection[storage.id] = ""
-    end
-    @row_collection[:item_quantity] = 0
-    @row_collection[:item_id] = nil
-  end
-  helper_method :new_storage_collection
-
-  def update_storage_collection(item)
-    @row_collection[item.storage_id] = item.quantity
-    @row_collection[:item_id] = item.id
-    @row_collection[:item_name] = item.name
-    @row_collection[:item_category] = item.category
-    @row_collection[:item_barcode_count] = item.barcode_count
-    @row_collection[:item_quantity] += item.quantity.nil? ? 0 : item.quantity
-  end
-  helper_method :update_storage_collection
 
   private
 
+  # Need to trick autoload to find Query::ItemsByStorageCollection(AndQuantity)
```

Comment:
:sob: This was the most reliable, fast workaround I could come up with - I don't think I understand Rails autoloading in enough depth to have a better solution. Suggestions appreciated!